src/elements/accordion/src/index.tsx

```diff
@@ -3,6 +3,7 @@
 import React, { useContext, useState } from 'react'
 import { jsx, Styled, useThemeUI } from 'theme-ui'
 import { Glyph, Icon } from '@uswitch/trustyle.icon'
+import { ImgixImage } from '@uswitch/trustyle.imgix-image'
```

Comment:
I think drone is breaking because you need to add this to the package.json

Reply:
Just fixed, drone is passing now